### PR TITLE
share quiz modal added

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
share quiz modal added

image
image

### CODE REVIEW

File: src/web-app/Component/SharePage/SharePage.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  return(
+        <div className="flex-col row-gap-2rem flex-justify-content-center flex-align-item-center">
+            <section>
+                <label> Share Quiz Page </label>
+            </section>
+            <section className="flex-row col-gap-1rem flex-justify-content-center flex-align-item-center">
+                <i class="fa-solid fa-envelope share-icon"></i>
+                <i class="fa-brands fa-twitter-square share-icon"></i>
+                <i class="fa-brands fa-facebook-square share-icon"></i>
+                <i class="fa-brands fa-linkedin share-icon"></i>
```
Comment (krgarima) [inline src/web-app/Component/SharePage/SharePage.jsx:13]: change class to className

File: src/web-app/Pages/rules/Rules.jsx [hunk 2]
```diff
@@ -27,19 +29,22 @@ const Rules =()=>{
...
                         <ul className='flex-col row-gap-1rem' >
                             <li> <i class="fa-solid fa-chevron-right"></i> Each question has 4/3/2 options, select the one correct answer. </li>
                             <li> <i class="fa-solid fa-chevron-right"></i> For one correct answer, you get <b>1 points</b>. </li>
                             <li> <i class="fa-solid fa-chevron-right"></i> No negative marking for incorrect answers. </li>
                             <li> <i class="fa-solid fa-chevron-right"></i> You have <b>5 mins</b> to solve <b>8 qestions</b>. </li>
                             <li> <i class="fa-solid fa-chevron-right"></i> All questions are compulsory. </li>
-                            <li> Let's begin!! </li>
+                            <li className='quiz-card-share'> 
+                               <label> Let's begin!! </label> 
+                               <i class="fa-solid fa-share" onClick={()=>{set_showmodal(!showmodal)}}></i>
```
Comment (krgarima) [inline src/web-app/Pages/rules/Rules.jsx:44]: here too

**Review by krgarima — COMMENTED** [review]
Nice UI. Good work. It would be better if I could see the site as well.

**Review by poojakelkar — APPROVED** [review]
Nice UI, good folder structure

**Pull request merged**